gradle/shipkit.gradle

```diff
@@ -0,0 +1,27 @@
+shipkit {
+   gitHub.repository = "linkedin/dynamometer"
+
+   //TODO generate and use your own read-only GitHub personal access token
+   gitHub.readOnlyAuthToken = "76826c9ec886612f504d12fd4268b16721c4f85d"
+
+   //TODO generate GitHub write token, and ensure your Travis CI has this env variable exported
```

Comment:
Can you remove the `TODO` on the environment variables and change it to just provide instructions on how to set these (a pointer to https://github.com/mockito/shipkit/blob/master/docs/getting-started.md is fine)